index.html

```diff
@@ -1,123 +1,95 @@
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/twittertwitter.png"
-                  alt="Twitter logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img class="img" src="Images/Linkedin.png" alt="LinkedIn logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/mediummedium.png"
-                  alt=" Medium logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/githubgithub.png"
-                  alt="Github logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/angellistangelist.png"
-                  alt="Angellist logo"
-              /></a>
-            </li>
-          </ul>
-        </div>
+    </nav>
+    <section class="front-page">
+      <div class="space"></div>
+      <h1 class="header">Hey there. I'm Cisco.</h1>
+      <h1 class="article">I'm a software developer.</h1>
+      <div>
+        <p class="p">
+          I can help you build a product, feature or website. Look through
+          some of my work and experience! If you like what you see and have a
+          project you need developed, don't hesitate to contact me.
+        </p>
+      </div>
+      <div>
         <div class="space"></div>
-        <div>
-          <img
-            class="drop-down"
-            src="Images/drop-down.png"
-            alt="drop-down button"
-          />
-        </div>
-      </section>
-    </main>
-    <section class="rest">
-      <h2 class="works" id="recent">My Recent Works</h2>
-      <div class="indicator">
-        <img id="line" src="Images/Indicator.png" alt="fancyLine"/>
       </div>
-      <div class="row">
+      <div class="images">
+        <ul>
+          <li>
+            <a href="#"><img class="img" src="Images/twittertwitter.png" alt="Twitter logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img class="img" src="Images/Linkedin.png" alt="LinkedIn logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img class="img" src="Images/mediummedium.png" alt=" Medium logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img class="img" src="Images/githubgithub.png" alt="Github logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img class="img" src="Images/angellistangelist.png" alt="Angellist logo" /></a>
+          </li>
+        </ul>
+      </div>
+      <div class="space"></div>
+      <div>
+        <img class="drop-down" src="Images/drop-down.png" alt="drop-down button" />
+      </div>
+    </section>
+  </main>
+  <section class="rest">
```

Comment:
I suggest you put your cards in a container and use display grid or change this section to use grid. This section here that contains your projects should use grid to position your individual projects.

In order to pass this milestone, you need to use `display: grid`  to order the cards and `display: flexbox` to order cards contents for both the mobile version and desktop version
<img width="1680" alt="Screenshot 2022-09-30 at 04 41 47" src="https://user-images.githubusercontent.com/48546992/193171992-9d20e163-bb7e-4b0a-b5d1-3e206213550a.png">

Reply:
Done